app-sso/getting-started/appsso-overview.hbs.md

```diff
@@ -78,3 +80,207 @@ applications may verify their validity using the AuthServer's public keys.
 ## <a id='next-steps'></a> Next steps
 
 - [Provision an AuthServer](provision-auth-server.md)
+
+## Provision an AuthServer
```

Comment:
**[issue]**: if using anchors above, make sure to include an `a` tag for each referred anchor.

Example:

```
## <a href='provision-an-authserver'></a> Provision an AuthServer
```

Then you can refer to it with `#provision-an-authserver`.

Otherwise docs engine cannot find the anchor point. Wish this was simpler and it eventually may be fixed.

Reply:
Oh, I thought that worked without that. Thank you!